### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by andogro**
I made some basic changes to enhance the Readme and added a Contributing and Code of Conduct file. Also fixed a typo in the sync_with_infura script. 
I will add some additional info about the Umbrella project structure next and potentially info about keccack and other technical details.  Let me know what else you'd like to see.

A couple of questions:

1. Is it just the Mana project, or Exthereum Mana
2. In the contributing file are there other considerations you would like to add? Should contributors run mix credo for example before submitting a pr, or any other specifics that would make your lives easier?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  1. If you can isolate the bug, fix it in a separate PR.
+  2. If the fix depends on your other commits, add it in a separate commit to the same PR.  
+
+    In either case, try to write a regression test that fails because of the bug but passes with your fix.
+
+
+### Issues
+Creating and discussing [Issues](https://github.com/poanetwork/mana/issues) provides significant value to the project. If you find a bug you can report it in an Issue.     
+
+### Pull Requests
```
Comment (germsvel) [inline CONTRIBUTING.md]: 👍 nice to have this in there. I wonder if we should ask people to open an `issue` before opening `PR`s? Or do you think PRs without issues make sense too?

File: README.md [hunk 2]
```diff
@@ -1,50 +1,56 @@
-# Mana
+# Mana Ethereum Implementation
```
Comment (germsvel) [inline README.md]: Is this the full name of the project?

File: README.md [hunk 3]
```diff
@@ -1,50 +1,56 @@
...
 # Dependencies
 
  * Elixir ~> 1.6.5
  * Rust ~> 1.26.0 (as a dependency of [Rox](https://github.com/urbint/rox))
 
+
 # Installation
 
-* Clone repo with submodules (so you can get the shared tests),
+* Clone repo with submodules (to access the Ethereum common tests)
```
Comment (germsvel) [inline README.md]: 👍

File: README.md [hunk 4]
```diff
@@ -70,3 +76,38 @@ cd apps/blockchain && mix test test/blockchain/state_test.exs
...
+* [Ethereum yellow paper](https://ethereum.github.io/yellowpaper/paper.pdf)(ETHEREUM: A SECURE DECENTRALISED GENERALISED TRANSACTION LEDGER
+BYZANTIUM VERSION )
+
+* [Message Calls in Ethereum](http://www.badykov.com/ethereum/2018/06/17/message-calls-in-ethereum/)
+
+Additional Ethereum Implementations
+
+* [Parity](https://github.com/paritytech/parity-ethereum)
+* [Geth](https://github.com/ethereum/go-ethereum/)
+* [EthereumJS](https://github.com/ethereumjs/ethereumjs-vm)
```
Comment (germsvel) [inline README.md]: Not sure if we want to include `py-evm` in here? https://github.com/ethereum/py-evm

File: README.md (hunk 2 above)
Comment (andogro) [reply README.md]: Not officially - I added to clarify what Mana is in the title of the repo but open to any other suggestions.

File: README.md (hunk 4 above)
Comment (andogro) [reply README.md]: Sure, I will add.

File: README.md (hunk 2 above)
Comment (igorbarinov) [reply README.md]: there is a new trend in naming `https://github.com/paritytech/parity-ethereum` following go-ethereum 
could be `mana-ethereum`

**Comment by masonforest** [comment]
In answer to your questions:

>  1. Is it just the Mana project, or Exthereum Mana

I think it's just the Mana project. We're still in the process of removing left over references to Exthereum.

> 2.  In the contributing file are there other considerations you would like to add? Should contributors run mix credo for example before submitting a pr, or any other specifics that would make your lives easier?

Here's a outline @ayrat555 and @germsvel feel free to add to it:

Before submitting a PR:
1. Add a test for the new or updated functionality. For tests that are simple and would make sense in documentation add a doctest. For more complicated tests add a regular test. If you're not sure which type to add ask in Gitter.

2. Run `mix format` to format your code
3. Run `mix credo` and fix any errors and warnings

**Comment by masonforest** [comment]
Looks good to me. Thanks @andogro !

File: CONTRIBUTING.md (hunk 1 above)
Comment (andogro) [reply CONTRIBUTING.md]: I think PR without issue is ok for now - we can amend later if it becomes problematic.

**Pull request merged**